### DISCUSSION TITLE
How to implement scalability for a 3D model in an ARScene by incorporating a  button child node with 3D model dedicated to scalability.

### DISCUSSION THREAD

**Discussion by UmerHamza** [sceneview/sceneview#515] [Q&A] [answered]

**Comment by ridcore** [comment, marked as answer]
You must activate the isScaleEditable property of the modelNode and with the button execute modelNode.setScale(1f) to increase the size.

[ModelNode Class](https://sceneview.github.io/api/sceneview-android/sceneview/io.github.sceneview.node/-model-node/index.html)

**Comment by UmerHamza** [comment]
I am looking for to change size of model in ARScene  real environment. i don't want to fix it before.
